src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const NUMBER_OF_DAYS = days;
+  const DAILY_COST = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const BASE_PRICE = NUMBER_OF_DAYS * DAILY_COST;
```

Comment:
It is customary to name only static constants with capital letters, in which there will be no calculations, so here it would be worthwhile to name the constant using camelCase (`basePrice`).

Reply:
Done.